gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Driver/GremlinClientTests.cs

```diff
@@ -167,6 +168,25 @@ public class GremlinClientTests
             }
         }
 
+        [Fact]
+        public async Task ShouldReturnResponseAttributes()
+        {
+            var gremlinServer = new GremlinServer(TestHost, TestPort);
+            using (var gremlinClient = new GremlinClient(gremlinServer))
+            {
+                var expectedResult = new List<int> { 1, 2, 3, 4, 5 };
+                var requestMsg = $"{nameof(expectedResult)}";
+                var bindings = new Dictionary<string, object> { { nameof(expectedResult), expectedResult } };
+
+                var response = await gremlinClient.SubmitAsync<int>(requestMsg, bindings);
```

Comment:
It doesn't matter which request you're submitting here. So why not make it as simple as possible? You can for example use `_requestMessageProvider.GetDummyMessage()`. Then you don't need the variables `expectedResult`, `requestMsg`, and `bindings`.

Reply:
Done